proposal/pachyderm.md

```diff
@@ -0,0 +1,246 @@
+          name: jupyterhub
+        - kustomizeConfig:
+            overlays:
+              - additional
+            repoRef:
+              name: manifests
+              path: jupyterhub/notebook-images
+          name: notebook-images
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: ceph/object-storage/scc
+          name: ceph-nano-scc
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: ceph/object-storage/nano
+          name: ceph-nano
+        - kustomizeConfig:
+            parameters:
+              - name: namespace
+                value: openshift-operators
+              - name: pachyderm_version
+                value: 0.0.8
+            repoRef:
+              name: manifests
+              path: odhpachyderm/operator
+          name: odhpachyderm-operator
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: odhpachyderm/deployer
+          name: odhpachyderm-deployer
+  ```
+- KfDef for Pachyderm with Other Storage
+  ~~~
+    apiVersion: kfdef.apps.kubeflow.org/v1
+    kind: KfDef
+    metadata:
+      name: opendatahub
+      namespace: opendatahub
+    spec:
+      applications:
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: odh-common
+          name: odh-common
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: odh-dashboard
+          name: odh-dashboard
+        - kustomizeConfig:
+            parameters:
+              - name: s3_endpoint_url
+                value: s3.odh.com
+            repoRef:
+              name: manifests
+              path: jupyterhub/jupyterhub
+          name: jupyterhub
+        - kustomizeConfig:
+            overlays:
+              - additional
+            repoRef:
+              name: manifests
+              path: jupyterhub/notebook-images
+          name: notebook-images
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: ceph/object-storage/scc
+          name: ceph-nano-scc
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: ceph/object-storage/nano
+          name: ceph-nano
+        - kustomizeConfig:
+            parameters:
+              - name: namespace
+                value: openshift-operators
+              - name: pachyderm_version
+                value: 0.0.8
+            repoRef:
+              name: manifests
+              path: odhpachyderm/operator
+          name: odhpachyderm-operator
+        - kustomizeConfig:
+            parameters:
+            - name: storage_secret
+              value: pachyderm-aws-secret
+            repoRef:
+              name: manifests
+              path: odhpachyderm/deployer
+          name: odhpachyderm-deployer  
+  ~~~
+
+**Upstream Pachyderm Components**
```

Comment:
@Jooho  Do you know if these images are built from the Community Edition source? We would want to make sure any container images can be traced back to their build files.
 
I think most OCP clusters will have an authenticated pull secret but I don't know if "free" OCP clusters like CodeReadyContainers or OKD will be able to deploy this without the cluster-admin user adding image pull secrets for a (free?) RH user account

Reply:
@LaVLaS As far as I know, OKD,CRC can install certified operators but that is not supported for production. (I tested CRC today but I didn't test OKD). Regarding Community Edition source, yes it is built with the public source.

Reply:
@LaVLaS odhseldon is also using certified operator so I believe it should be ok.

Reply:
That makes sense. Use of any operator (certified or community) is fine as long as it is "freely" available and can be recreated from the open source version.